pytest-selenium/regions/my_highlights.py

```diff
@@ -397,11 +400,11 @@ def color_filters(self) -> MyHighlights.FilterBar.ColorFilters:
 
             """
             filter_menu_root = self.find_element(
-                *self._color_filters_locator)
+                *self._color_dropdown_locator)
             return self.ChapterFilters(self, filter_menu_root)
```

Comment:
the property return type should be `ChapterFilters` instead of `bool` and you may want to rename the function as it is no longer returning a yes/no value

Reply:
@gregfitch can we have a call to go over this and the below comment. I dont see any need to change the return type